plugins/chat/assets/javascripts/discourse/initializers/chat-sidebar.js

```diff
@@ -424,7 +423,19 @@ export default {
             }
 
             get suffixValue() {
-              return this.channel.tracking.unreadCount > 0 ? "circle" : "";
+              return this.channel.tracking.unreadCount > 0 ||
+                (this.chatService.activeChannel?.id !== this.channel.id &&
+                  this.channel.unreadThreadsCountSinceLastViewed > 0)
+                ? "circle"
+                : "";
+            }
+
+            get suffixCSSClass() {
```

Comment:
We now have threads within DMs, so treating it in the same way as public channel threads. When it is tracked we should show a blue badge. When there is a mention, regular DM message or watched thread then it should be a green badge.